README.md

```diff
@@ -35,25 +35,23 @@ another library.
 ```rust
```

Comment:
I think if you add the following to `lib.rs` (and add `ignore here`), then the example will be checked by CI, and this sort of oversight won't happen again.
```rust
#[cfg(doctest)]
#[doc = include_str!("../README.md")]
extern "C" {}
```